### DISCUSSION TITLE
무조건 로그인실패뜨네요 ..

### DISCUSSION THREAD

**Discussion by difmwng3** [storycraft/node-kakao#396] [Q&A]
![image](https://user-images.githubusercontent.com/55140951/121813931-5c98f700-cca9-11eb-83f5-f6168e384df7.png)

관련 파일 설치 다하고햇는데
뭐가 미스가 난건지 ..

예제도없고 도와좀주셔요 ㅠ

**Comment by generalxd** [comment]
암호틀린거같습니다.

**Reply by difmwng3** [reply]
100% 1000% 동일합니다 .
제 pc 에서 구동하는 제 이멜과 비밀번호를 입력 하였습니다 ㅠ.ㅠ

**Comment by JellyBrick** [comment]
<table>
<tr>
	<td><b>#</b>
	<td><b>발생 이유</b>
	<td><b>추가 설명</b>
<tr>
	<td>30
	<td>일부 필드 값이 잘못 됨
	<td>(e.g. 비밀번호가 잘못됨, 아이디가 없음, X-VC 코드 에러[모바일로 로그인 차단 메세지 전송])
</table>

비밀번호 / 아이디 확인 부탁드립니다.

**Reply by difmwng3** [reply]
100% 1000% 동일합니다 .
제 pc 에서 구동하는 제 이멜과 비밀번호를 입력 하였습니다 ㅠ.ㅠ

**Reply by JellyBrick** [reply]
> 100% 1000% 동일합니다 .
> 제 pc 에서 구동하는 제 이멜과 비밀번호를 입력 하였습니다 ㅠ.ㅠ

혹시 기기 인증 하셨나요?

**Reply by difmwng3** [reply]
로그인 하기 전에 
예제파일안에서 최초로그인 하시는거 말씀하시는걸까요 ?
그거라면 add.ts 가 그 소스입니다
